lang/step.md

```diff
@@ -384,7 +394,10 @@ impl<M: Memory> Machine<M> {
         }
 
         // Advance the PC for this stack frame.
-        self.cur_frame_mut().next = (next_block, BigInt::zero());
+        self.mutate_cur_frame(|frame| {
+            frame.next = (next_block, BigInt::zero());
```

Comment:
Might be worth factoring this into a `frame.jump_to_block()` so that one cannot forget to also update the next_stmt to 0.

Reply:
Yeah, that sounds good.
I think as `jump_to_block` would only be used as `self.mutate_cur_frame(|frame|) frame.jump_to_block(_))`
it might make sense to make it `machine.jump_to_block(_)`?

Reply:
Hm, conceptually it is a frame action... I think defining it on the frame is better. It exposes structurally that that is the only piece that changes.

Reply:
okay, is changed now.